repository/l.json

```diff
@@ -673,6 +673,18 @@
 				}
 			]
 		},
+		{
+			"name": "LCOV",
+			"details": "https://gitlab.com/aquator/sublime-lcov",
+			"readme": "https://gitlab.com/aquator/sublime-lcov/-/raw/master/README.md",
```

Comment:
```suggestion
```

Since that's just the default readme location, you can omit it.

Reply:
Thanks, did not know that, I copied from another gitlab based plugin.
Removed the readme line.